ppr-ui/src/App.vue

```diff
@@ -20,16 +20,16 @@
         :show-login-menu="false"
       />
 
-    <!-- Alert banner -->
-    <v-alert
-      tile dense
-      type="warning"
-      v-if="bannerText">
-      <div v-html="bannerText" class="mb-0 text-center colour-dk-text"></div>
-    </v-alert>
-
     <div class="app-body">
       <main>
+        <sbc-system-banner
```

Comment:
I can't make v-alert works in ppr-ui. I bring back the sbc-system-banner.

Reply:
It could be our Vuetify version.

Reply:
Ok, I will leave it. When you upgrade to VUE3, you may review this.

Reply:
Definitely 👍